base/database/testing.go

```diff
@@ -312,3 +314,56 @@ func UpdateSystemAdvisoriesWhenPatched(t *testing.T, systemID, accountID int, ad
 		Update("when_patched", whenPatched).Error
 	assert.Nil(t, err)
 }
+
+func CreateBaseline(t *testing.T) int {
```

Comment:
I would improve this helper method with option to add systems directly (inventoryIDs []string)